src/handlers/cell.ts

```diff
@@ -216,6 +256,18 @@ export class CellManager implements IDisposable {
     });
   }
 
+  private getBreakpointsFromEditor(cell: CodeCell): ILineInfo[] {
```

Comment:
This method was removed in https://github.com/jupyterlab/debugger/pull/165, but we still need to retrieve the breakpoints from the editor when the content of the cell changes.

Reply:
cc @JohanMabille 

It looks like we need to keep this method, or we might end up in situations such as the one described here: https://github.com/jupyterlab/debugger/issues/172#issue-521721030

Also it feels more natural to have the breakpoints stay where they are when editing a cell.

What do you think?

Reply:
Indeed this methods should be added back. I removed it when I refactored the logic for setting breakpoints because it was not called anymore, but it should definitely be reintroduced.

Reply:
ok good :+1: